compiler/optimizer/Inliner.cpp

```diff
@@ -5689,6 +5689,8 @@ TR_CallSite::addTarget(TR_Memory* mem, TR_InlinerBase *inliner, TR_VirtualGuardS
 
    addTarget(result);
 
+   _comp->validateImplementer(implementer);
```

Comment:
seems strange to ignore the return value...

Reply:
I know why; it's a bad reason but I remember why I ignored this. It's because the code locations that addTarget don't really have a fall back path; once it's gotten to calling `addTarget`, it just assumes adding to the list of targets should just work; getting the concept of "oh we might not want to do so for some reason" requires a bit of work; I'll open an issue to track this.

Reply:
FWIW, to give OpenJ9 as an example, because we're validating the implementer which is a method symbol, there is no way for a validation to fail (a class symbol validation can fail if we can't remember the class). However, you're right in that in general, we shouldn't ignore this.

Reply:
Can you at least assert on true return value?